data-integration/main.bal

```diff
@@ -0,0 +1,102 @@
+import data_integration.store;
+import ballerina/http;
+import ballerina/persist;
+
+configurable int port = 9090;
+
+service /taskmanager on new http:Listener(port) {
```

Comment:
Doesn't feel like an appropriate path to me. Shall we avoid this? We can skip it altogether IMO.